docs/internal/continuous-profiler.md

```diff
@@ -0,0 +1,255 @@
+# Continuous profiler
+
+> [!IMPORTANT]  
+> Continuous profiler is an experimental feature. It will be subject to change,
+> when <https://github.com/open-telemetry/oteps/pull/239> or <https://github.com/open-telemetry/oteps/pull/237>
+> are merged.
+
```

Comment:
```suggestion

Continuous profiler collects stack traces from the processes for two type of events

* periodically, for all threads -- see thread sampling,
* for memory allocation events -- see memory allocation.

Stack traces can be exported to observability back end that supports profiling.

```

Reply:
Nice! A small rewording:

```suggestion
The continuous profiler collects stack traces from the processes for two type of events:
	
	* Periodically, for all threads. See Thread sampling.
	* Memory allocation events. See Memory allocation.
	
You can export stack traces to any observability back end that supports profiling.
```

If you'd add anchor links in each bullet point, that'd be fantastic!